src/main/reactjs/public/i18n/koodisto/langs/koodisto_langs_sv-SE.json

```diff
@@ -150,6 +150,7 @@
         "SC": "sardiska",
         "SD": "sindhi",
         "SE": "samiska",
+        "SEPO": "samiska",
```

Comment:
Eikö koodistossa ole oikeita käännöksiä eri saamen murteille? Yksi vaihtoehto voisi olla lisätä ne tästäkin huolimatta näihin lähes staattisiin filuihin.

Reply:
Jos näin tehdään niin käännökset olisivat vissiin seuraavat:
```
"SEIN": "enaresamiska",
"SEKO": "skoltsamiska",
"SEPO": "nordsamiska"
```

Reply:
Ei ole, sieltä tuli mitä tuli. Sanoisin että emme ala korjaamaan koodiston puutteita, koska ne pitäisi korjata koodistoon, ja jos jokainen koodiston käyttäjä korjaa ne itse ei asiaa korjata itse koodistoon. Jos ei muuta ole, niin mergetetään tällaisenaan ja tehdään tiketti että koodistosta puuttuu saamen kielten nimet ruotsiksi ja englanniksi.

Reply:
Jep, näinhän sen pitäisi mennä.